src/features/search/components/sections/DateHour/DateHour.tsx

```diff
@@ -22,7 +22,11 @@ const buildDateString = (date: SearchState['date']) => {
   }
 }
 
-export const DateHour = () => {
+type Props = {
+  onClose?: VoidFunction
+}
+
+export const DateHour: FunctionComponent<Props> = ({ onClose }) => {
```

Comment:
```suggestion
export const DateHour = ({ onClose }: Props) => {
```

il vaut mieux laisser TypeScript `infer` le retour

Reply:
ça me va après cette façon de faire est quand même pas mal utilisée (enfin parmi d'autres).
Il faudrait peut être mettre à l'ordre du jour d'une synchro front pour tenter d'harmoniser (si ça n'a pas déjà été fait, il faudrait que je re-check les docs)

Reply:
C'est la nouvelle best practice (nouvelle puis 2 ans je crois)